lib/tasks/rake_helpers/dump_file_writer.rb

```diff
@@ -54,7 +54,7 @@ def add_values(attribute_names)
   end
 
   def prepare_sql
-    table.compile_insert(data).to_sql.gsub('"', '') + ';'
+    table.compile_insert(data).to_sql + ';'
```

Comment:
A word of warning. I tried to find out what was going on here, and how the string is quoted, and the entire [documentation for the `compile_insert` method](https://www.rubydoc.info/docs/rails/Arel/Crud) says:

> FIXME hopefully we can remove this

My guess is that this is an internal method and not intended to be used.

It might be possible to do something more future-proof like;

```ruby
model.create(data).to_sql
```

(but this is beyond the scope.of this PR)

Reply:
👍  feel free to change, test locally and PR. ultimate test is to run against prod though though so is time consuming to work on